scripts/hist_importer.py

```diff
@@ -0,0 +1,142 @@
+             'ed: http://kb.mozillazine.org/Profile_folder'\
+             '\n\tChrome: Is named "History", and can be found at the respec'\
+             'tive User Data Directory. Check this link for where it is locat'\
+             'ed: https://chromium.googlesource.com/chromium/src/+/master/'\
+             'docs/user_data_dir.md\n\n'\
+             'Example: $this_script.py -b firefox -s /Firefox/Profile/places.'\
+             'sqlite -d /qutebrowser/data/history.sqlite'
+    parsed = argparse.ArgumentParser(
+        description=description, epilog=epilog,
+        formatter_class=argparse.RawTextHelpFormatter
+    )
+    parsed.add_argument('-b', '--browser', dest='browser', required=True,
+                        type=str, help='Browsers: {firefox, chrome, safari}')
+    parsed.add_argument('-s', '--source', dest='source', required=True,
+                        type=str, help='Source: fullpath to the sqlite data'
+                        'base file from the source browser.')
+    parsed.add_argument('-d', '--dest', dest='dest', required=True, type=str,
+                        help='Destination: The fullpath to the qutebrowser '
+                        'sqlite database')
+    return parsed.parse_args()
+
+
+def open_db(data_base):
+    """Open connection with database."""
+    try:
+        conn = sqlite3.connect(data_base)
+        return conn
+    except Exception as any_e:
+        print('Error: {}'.format(any_e))
+        raise('Error: There was some error trying to to connect with the [{}]'
+              'database. Verify if the filepath is correct or is being used.'.
+              format(data_base))
+
+
+def extract(source, query):
+    """Performs extraction of (datetime,url,title) from source."""
+    try:
+        conn = open_db(source)
+        cursor = conn.cursor()
+        cursor.execute(query)
+        history = cursor.fetchall()
+        conn.close()
+        return history
+    except Exception as any_e:
+        print('Error: {}'.format(any_e))
+        print(type(source))
+        raise('Error: There was some error trying to to connect with the [{}]'
+              'database. Verify if the filepath is correct or is being used.'.
+              format(str(source)))
+
+
+def clean(history):
+    """Receives a list of records:(datetime,url,title). And clean all records
+    in place, that has a NULL/None datetime attribute. Otherwise Qutebrowser
+    will throw errors. Also, will add a 4rth attribute of '0' for the redirect
+    field in history.sqlite in qutebrowser."""
+    nulls = [record for record in history if record[0] is None]
+    for null_datetime in nulls:
+        history.remove(null_datetime)
+    history = [list(record) for record in history]
+    for record in history:
+        record.append('0')
+    return history
+
+
+def insert_qb(history, dest):
+    """Given a list of records in history and a dest db, insert all records in
+    the dest db."""
```

Comment:
Same things as above - please have a summary in imperative mood on the first line.